CONTRIBUTING.md

```diff
@@ -5,17 +5,51 @@ To get up and running, install the dependencies and run the tests:
 ```bash
 yarn
 yarn test
-```
 
-Run `yarn prettier path/to/file.php` to format a single file.
+# format a single file
+yarn prettier path/to/file.php`
+
+# run a single test
```

Comment:
I'd either clarify or go into detail what running the `kitchen_sink` test actually *does*. I don't know it, and I cannot infer it from this comment. Quite the contrary — "run a single" test sounds to me like "this is how you run one specific test of your choice", which this command obviously does not do.

If this command does what I think it does, something like `# run a quick allrounder test against Prettier` may be more fitting.

Reply:
The only thing that's special about "kitchen_sink" is that it's not specific to a topic, which makes it a good candidate if one just wants to run a very specific test. I removed the reference to kitchen_sink, because it might be something that only I am using that way :wink:

Reply:
I didn't dislike it though. Having a quick allrounder test for checking whether you broke something fundamental sounds valuable to me — but since I haven't done any coding on the plugin itself, I cannot tell.

When do you typically use it?

Reply:
I usually replace the entire content of kitchen_sink with the code I'm currently playing around with. So the actual content of kitchen_sink.php doesn't even add much value. That's why I prefer to remove it.

Reply:
Oh. Yeah that's probably a bit too convulted to put it into the docs. 😁

I think the "format a single file" command should be sufficient then.

Reply:
Right :wink: 

Let's leave it in since it was asked for in https://github.com/prettier/plugin-php/issues/1053#issuecomment-570082839